slack_bolt/workflows/step/async_step_middleware.py

```diff
@@ -13,9 +12,8 @@
 class AsyncWorkflowStepMiddleware(AsyncMiddleware):
     """Base middleware for step from app specific ones"""
 
-    def __init__(self, step: AsyncWorkflowStep, listener_runner: AsyncioListenerRunner):
```

Comment:
While this is a public class, I am sure that no developer relies on this inconvenient constructor, and more importantly steps from apps is no longer available for Workflow Builder. Thus, this minor breaking change is okay but others feel we should avoid this, I am happy to revert it.